site/content/en/contribute/howitworks/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Writing Code"
+linkTitle: "Writing Code"
+type: docs
+weight: 40
+description: >
+    How to modify Kustomize
+---
+
+{{< alert color="success" title="Info" >}}
+To build kustomize using the locally modified modules, `replace` statements must be added to
```

Comment:
We now use [go workspace](https://github.com/kubernetes-sigs/kustomize/pull/4692) mode, so we no longer need replace statements.

Reply:
okie! made those changes!